### ISSUE TITLE
Admod has stopped working (AdMobRewarded)

### ISSUE THREAD

**Issue by georgeMorales** [expo/expo#12858]
### Summary

I have been building an app for about 5 months and using Admob (AdMobRewarded) in test mode on iOS and Android, so far AdMod's AdMobRewarded has worked well but suddenly it no longer works today.
I also have another app in test mode sdk 40 with AdMobRewarded that a couple of days ago the last time I tried it worked, today it does not work either.
Expo sdk 41managed
console log error: Could not show ad [Error: No ad config.]

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

Android, iOS

### Environment

Expo CLI 4.4.4 environment info:
    System:
      OS: Linux 4.15 Ubuntu 16.04.7 LTS (Xenial Xerus)
      Shell: 4.3.48 - /bin/bash
    Binaries:
      Node: 10.16.0 - /usr/local/bin/node
      Yarn: 1.21.1 - /usr/bin/yarn
      npm: 6.13.4 - /usr/local/bin/npm
      Watchman: 4.5.0 - /usr/local/bin/watchman
    SDKs:
      Android SDK:
        API Levels: 23, 25, 26, 27, 28, 29
        Build Tools: 23.0.1, 23.0.2, 23.0.3, 25.0.0, 25.0.1, 25.0.2, 25.0.3, 26.0.0, 26.0.1, 26.0.2, 26.0.3, 27.0.3, 28.0.3, 29.0.2
        System Images: android-23 | Intel x86 Atom, android-23 | Intel x86 Atom_64, android-23 | Google APIs Intel x86 Atom, android-24 | Google APIs Intel x86 Atom, android-25 | Google APIs Intel x86 Atom, android-26 | Google APIs Intel x86 Atom, android-26 | Google APIs Intel x86 Atom_64, android-28 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 4.1 AI-201.8743.12.41.6953283
    npmPackages:
      expo: ^41.0.0 => 41.0.1 
      react: 16.13.1 => 16.13.1 
      react-dom: 16.13.1 => 16.13.1 
      react-native: https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz => 0.63.2 
      react-native-web: ~0.13.12 => 0.13.18 
    npmGlobalPackages:
      expo-cli: 4.4.4
    Expo Workflow: managed

### Reproducible demo or steps to reproduce from a blank project

```
    const adUnitID = Platform.select({
      ios: 'ca-app-pub-3940256099942544/1712485313',
      android: 'ca-app-pub-3940256099942544/5224354917',
    });
```

**Comment by brentvatne** [comment]
it sounds like something changed on google's side or on your google app configuration.

you should try updating to the latest expo sdk, we updated admob.

**Comment by georgeMorales** [comment]
@brentvatne I upgraded to expo sdk 41 a few weeks ago and everything was fine.

**Comment by brentvatne** [comment]
@georgeMorales - i can say with certainty that we did not change any of our code that is related to this, and certainly not in your project on your machine or deployed to stores, and so if we isolate that the only thing that is left is something on google's side, perhaps with your account configuration

**Comment by georgeMorales** [comment]
@brentvatne Thanks for your help, it's weird because I've been working with the test account for a few months because I'm in development and it has always worked well, a few weeks ago I updated and made the new changes and everything worked fine the same as before ... but this morning it no longer . I suppose it will be something that google has changed from the account or I do not know what it could be.

**Comment by brentvatne** [comment]
@georgeMorales - maybe google support could help. sorry for the trouble, hopefully you can work it out!

**Comment by georgeMorales** [comment]
It was not solved
I have an app in production on Android for about 3 months generating ads but on IOS in production, admob does not work. I've tried everything and it doesn't work. On iOS in development it works fine and shows me the test ads but when I compile it no longer works. ERROR_CODE_NO_FILL, I have entered the bank account in admob, we have entered the bank details, it has been more than four months since we activated the IOS ad units, I have checked the ad unit codes a thousand times and nothing ...
The truth is that I'm desperate and I don't know what else to do or look at.
Hope SDK 43 comes out soon and fixes this issue.
